go.mod

```diff
@@ -57,23 +59,24 @@ require (
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/spiffe/go-spiffe v0.0.0-20190717182101-d8657cb50cae
+	github.com/spiffe/go-spiffe/v2 v2.0.0-alpha.2
 	github.com/spiffe/spire/proto/spire v0.0.0-20190723205943-8d4a2538e330
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
 	github.com/uber-go/tally v3.3.12+incompatible
-	github.com/zeebo/errs v1.2.0
+	github.com/zeebo/errs v1.2.2
 	go.uber.org/atomic v1.4.0
 	go.uber.org/goleak v0.10.0
 	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
-	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980
+	golang.org/x/net v0.0.0-20190620200207-3b0461eec859
 	golang.org/x/sys v0.0.0-20200122134326-e047566fdf82
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
-	golang.org/x/tools v0.0.0-20190618163018-fdf1049a943a
+	golang.org/x/tools v0.0.0-20191127201027-ecd32218bd7f
 	google.golang.org/api v0.6.0
 	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
-	google.golang.org/grpc v1.24.0
+	google.golang.org/grpc v1.27.1
```

Comment:
Looking at the unit test failures, looks like some gRPC error messages are now different?

Reply:
Oh, yep. They are much nicer :)